pypdf/_doc_common.py

```diff
@@ -1138,16 +1138,16 @@ def remove_page(
         Remove page from pages list.
 
         Args:
-            page: int / PageObject / IndirectObject
-                PageObject : page to be removed. If the page appears many times
-                only the first one will be removed
+            page: int / PageObject / IndirectObject.
```

Comment:
While at it, we might want to improve rendering to use a list for example: https://pypdf--2674.org.readthedocs.build/en/2674/modules/PdfReader.html#pypdf.PdfReader.remove_page